cypress/tools/fixture-generator.py

```diff
@@ -0,0 +1,59 @@
+import json
+import random
+import string
+import uuid
+from datetime import datetime, timedelta
+import os
+
+def random_string(length=8):
+    return ''.join(random.choices(string.printable, k=length))
+
+def random_underscores_string(length=8):
+    return ''.join(random_string(length) 
+                   + "_" + random_string(length) 
+                   + "_" + random_string(length))
+
+def random_date(start, end):
+    delta = end - start
+    random_days = random.randint(0, delta.days)
+    random_us = random.randint(1, 86400000000)
+    return start + timedelta(days=random_days, microseconds=random_us)
+
+start_date = datetime(2023, 7, 1)
+end_date = datetime(2024, 7, 1)
+
+data = []
+
+for _ in range(150):
+  date = random_date(start_date, end_date)
+
+  random_status = random.choice(['DRAFT', 'FINALIZED'])
+
+  uuid_value = str(uuid.uuid4())
+
+  template = {
+        "uri": f"https://api-test.ramanchada.ideaconsult.net/template/{random_string(21)}",
+        "uuid": uuid_value,
+        "METHOD": random_string(12),
+        "timestamp": date.isoformat(),
+        "PROTOCOL_CATEGORY_CODE": random_underscores_string(6),
+        "EXPERIMENT": random_string(12),
+        "template_name": random_string(42),
+        "template_status": random_status,
+        "template_author": random_string(42),
+        "template_acknowledgment": random_string(34)
+  }
+  data.append(template)
+
+current_directory = os.getcwd()
+
+file_name = "templates_generated.json"
+
+parent_directory = os.path.dirname(current_directory) + "/fixtures"
```

Comment:
I believe this should be `/fixtures/json`, maybe even `/cypress/fixtures/json` if we are going to run the script from the root project directory. If you are not sure, change it to `/cypress/fixtures/json` and run the script from the root directory of the project.